Exiled.API/Features/Player.cs

```diff
@@ -1224,6 +1224,12 @@ public bool HasItem(ItemType targetItem)
         /// <param name="reason">The disconnection reason.</param>
         public void Disconnect(string reason = null) => ServerConsole.Disconnect(GameObject, string.IsNullOrEmpty(reason) ? string.Empty : reason);
 
+        /// <summary>
+        /// Heals the player.
```

Comment:
Since this is limited to only healing them up to their max health, the summary should specify this to avoid confusion.

Reply:
Check it now and lemme know if it's ok.